tools/cell_census_builder/scripts/aws/mount_instance_storage.sh

```diff
@@ -9,6 +9,8 @@
 # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/raid-config.html
 #
 
+DEVICES="$@"
```

Comment:
super tiny nit - now that it takes params, maybe add a comment at top of file showing usage

Reply:
done!